src/components/classifier/SwipeClassifier.js

```diff
@@ -121,7 +120,7 @@ export class SwipeClassifier extends React.Component {
             onExpandButtonPressed={this.expandImage}
             subjectDisplayWidth={this.state.swiperDimensions.width}
             subjectDisplayHeight={this.state.swiperDimensions.height}
-            dimensionsLoaded={this.state.swiperDimensionsLoaded}
+            dimensionsLoaded={true}
```

Comment:
Is there a reason this was changed to always be true? Looks like this is the only component using `SwipeCard`, and there is still a conditional check for opacity in the component dependent on this prop.

Reply:
I'm going to clean up the `swiperDimensionsLoaded`, I don't think we need it.